### ISSUE TITLE
Steam using HTTPS not lancache

### ISSUE THREAD

**Issue by Glitch990** [lancachenet/sniproxy#16]
I can confirm this is still happening on Comcast. Steam is repeatedly defaulting to HTTPS. Fix #47 does not appear to be working anymore either. New docker images this morning.

_Originally posted by @Glitch990 in https://github.com/lancachenet/lancache-dns/issues/48#issuecomment-653650048_

**Comment by Glitch990** [comment]
Shutting down SNIProxy some steam games are cached, but most will fail to wait for the HTTPS request/responses

**Comment by Glitch990** [comment]
I came up with a temporary solution on my own, since this issue being well known is not being addressed.

I have figured out a "less than perfect" solution but it works surprisingly well. So I had to give up having my DNS resolver (Pfsense) do the forwarding, which is a big sacrifice. I am running SNIProxy, LANCache, and LANCache-DNS. I pointed my DHCP server at the LANCache-DNS for resolution. I have the SNIProxy, LANCache, and LANCache-DNS pointed to my router as the upstream. I then have two aliases in my firewall for the ComcastDNS addresses for both port 80 and 443 (I noticed that if any connection is made the subsequent connections try for HTTPS) and have those aliases blocked on the LAN side. This caches almost every game through LANCache (Battle Brothers does not cache).
